projects/plugins/jetpack/modules/subscriptions.php

```diff
@@ -461,6 +476,29 @@ public function subscription_comment_subscribe_setting() {
 		<?php
 	}
 
+	/**
+	 * Subscribe Modal Toggle.
+	 */
+	public function subscribe_modal_setting() {
+
+		$sm_enabled = get_option( 'sm_enabled', 1 );
+		?>
+
+		<p class="description">
+			<input type="checkbox" name="sm_enabled" id="jetpack-subscribe-modal" value="1" <?php checked( $sm_enabled, 1 ); ?> />
+			<?php
+			echo wp_kses(
+				__(
+					'Show a popup subscribe modal to readers.',
+					'jetpack'
+				)
+			);
```

Comment:
I don't know that we need `wp_kses` here. It could just be 

```suggestion
			esc_html_e( 'Show a popup subscribe modal to readers.', 'jetpack' );
```

Reply:
Yes, this is correct. A bit of copy/past going on here. Fixed.